EFAuditer/Auditer.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Audit.Core;
+using Newtonsoft.Json;
+
+namespace EFAuditer
+{
+    public static class Auditer
+    {
+        // TODO NTBS-218: Split this into a static class just containing the setup method to be called once, and a non-static auditer that can be injected as a singleton into the application that uses this
+        // Add Db read methods to fetch all data by e.g. Id, type etc.

```

Comment:
As you suggested, not part of the ticket - let's keep it simple. Once we get to the requirement of showing this data in the UI, we can look at fishing it out from the db :)